packages/performance-tests/test/TransformerRegression.test.ts

```diff
@@ -0,0 +1,127 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+
+/*
+ * Tests where we perform "identity" transforms, that is just rebuilding an entire identical iModel (minus IDs)
+ * through the transformation process.
+ */
+
+import "./setup";
+import { assert } from "chai";
+import * as path from "path";
+import * as fs from "fs";
+import { Element, IModelHost, IModelHostConfiguration, Relationship, SnapshotDb } from "@itwin/core-backend";
+import { Logger, LogLevel, PromiseReturnType, StopWatch } from "@itwin/core-bentley";
+import { IModelTransformer, TransformerLoggerCategory } from "@itwin/imodel-transformer";
+// import { TestBrowserAuthorizationClient } from "@itwin/oidc-signin-tool";
+import { getTestIModels } from "./TestContext";
+import { initOutputFile } from "./TestUtils";
+import { NodeCliAuthorizationClient } from "@itwin/node-cli-authorization";
+import { BackendIModelsAccess } from "@itwin/imodels-access-backend";
+import { IModelsClient } from "@itwin/imodels-client-authoring";
+import { TestBrowserAuthorizationClient } from "@itwin/oidc-signin-tool";
+import { Reporter } from "@itwin/perf-tools";
+import * as os from "os";
+
+async function sleep(ms: number) {
+  return new Promise((resolve) => {
+    setTimeout(resolve, ms);
+  });
+}
+const testCasesMap = new Map<string, any>();
+testCasesMap.set("identity transform", require("./identity-transformer"));
+// testCasesMap.set("prepare fork transform", require("./prepare-fork"));
+
+const loggerCategory = "Transformer Performance Tests Identity";
+const outputDir = path.join(__dirname, ".output");
+
+const setupTestData = async () => {
+  const logLevel = +process.env.LOG_LEVEL!;
```

Comment:
can you do like `+(process.env.LOG_LEVEL ?? "2")`?

Reply:
done

Reply:
my bad, I should have thought of:

```ts
const logLevel = process.env.LOG_LEVEL
  ? Number(process.env.LOG_LEVEL)
  : LogLevel.Warning;
```

That's much less confusing

Reply:
done